### PR TITLE
[website]: fix several typos and some phrasing

### PR DESCRIPTION
**Pull request by vlasebian**
I've been reading the docs on the website lately and found some typos. I thought if I am going through the docs, might as well do a PR to fix them.

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ curl -X POST http://127.0.0.1:8545 \
 This requires approval in Clef. Once approved, the following information is returned to the terminal:
 
 ```terminal
-{"jsonrpc":"2.0","id":1,"result":["0xca57f3b40b42fcce3c37b8d18adbca5260ca72ec"]}
+{"jsonrpc":"2.0","id":1,"result":["0xca57f3b40b42fcce3c37b8d18adbca5260ca72ec", "0xce8dba5e4157c2b284d8853afeeea259344c1653"]}
```
Comment (holiman) [inline docs/getting-started/index.md:413]:
```suggestion
{"jsonrpc":"2.0","id":1,"result":["0xca57f3b40b42fcce3c37b8d18adbca5260ca72ec"]}
```
please revert

**Review by holiman — COMMENTED** [review]
Thanks, looks good to me except for one thing

File: docs/getting-started/index.md (hunk 1 above)
Comment (vlasebian) [reply docs/getting-started/index.md]: done, I reverted this change

**Review by holiman — APPROVED** [review]
Thanks!

**Pull request merged**